others/Reverse_spiral_form_of _matrix.cpp

```diff
@@ -0,0 +1,81 @@
+/**
+ * @file
+ * @brief Given a 2D array print it in reverse spiral form
+ */
+#include <iostream>
+#include <stdlib.h>
+#include <algorithm>
+#include <vector>
+
+using namespace std;
```

Comment:
This is discouraged. Please strictly use `std::` or `using std::cin`, `using std::cout`, etc..